lib/Mock/Response.php

```diff
@@ -3,12 +3,17 @@
 namespace SAI\Mock;
 
 class Response implements \SAI\Response {
-  public $headers = array(), $cookies = array(), $cookies_deleted = array(), $output = '';
+  public $headers = array(), $cookies = array(), $cookies_deleted = array();
+  public $output = '', $response_code = null;
 
   public function setHeader($name, $value) {
     $this->headers[strtolower($name)] = $value;
   }
 
+  public function setResponseCode($code) {
+    $this->response_code = $code;
+  }
```

Comment:
The corresponding PHP function returns the previous response code.  I think we should do the same.  Defaulting to 200 then.